src/snowflake/cli/api/entities/application_package_entity.py

```diff
@@ -111,7 +113,14 @@ def action_deploy(
                 print_diff=True,
             )
 
-        # TODO Execute post-deploy hooks
+        if model.meta and model.meta.post_deploy:
+            self.execute_post_deploy_hooks(
+                console=ctx.console,
+                project_root=ctx.project_root,
+                post_deploy_hooks=model.meta.post_deploy,
+                package_name=package_name,
+                package_warehouse=model.meta.warehouse or ctx.default_warehouse,
```

Comment:
is that equivalent to self.package.warehouse?

Reply:
Yes, it points to `NativeAppProjectModel.package_warehouse` which has [the same logic](https://github.com/snowflakedb/snowflake-cli/blob/main/src/snowflake/cli/_plugins/nativeapp/project_model.py#L101-L109)